lib/metadata/index.js

```diff
@@ -125,6 +125,11 @@ Metadata.prototype.buildTokens = function (hosts) {
   this.primaryReplicas = primaryReplicas;
   //All the tokens in ring order
   this.ring = allSorted;
+  //Compute string versions as it's potentially expensive and frequently reused later
+  this.ringTokensAsStrings = [];
+  for (var i = 0; i < allSorted; i++) {
```

Comment:
Thanks! I think its a good step to have at least those 2 covered in the unit tests.